Documentation/conf.py

```diff
@@ -261,6 +261,27 @@
 # Try as hard as possible to find references
 default_role = 'any'
 
+# -- Options for Link Check output ----------------------------------------
+
+linkcheck_ignore = [
+    # Local links
+    'http://127.0.0.1',
+    'http://localhost',
+    'http://192.168.60.13',
+    'https://192.168.60.11',
+    
+    # URLs that are not reachable(invalid or test links)
+    'https://bookinfo.cilium.rocks',
+    'dns:query',
+
+    # Needs authentication
+    'https://github.com/cilium/cilium/projects/new',
+    'https://github.com/cilium/cilium/settings/branches',
+    'https://jenkins.cilium.io/job/',
+    'https://jenkins.cilium.io/view/'
+]
+
+linkcheck_anchors = False
```

Comment:
Does this skip checking the anchors? If so, why should we skip them, are they too hard to check?

Reply:
As mentioned above I tried to make work on the anchors and changed the one that were 
broken, but the `Reamde#anchor` even though they exist are throwing the warnings. 
```
/src/Documentation/gettingstarted/gettinghelp.rst:47: WARNING: broken link: https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md#listing-cilium-training (Anchor 'listing-cilium-training' not found)
/src/Documentation/contributing/testing/e2e_legacy.rst:829: WARNING: broken link: https://github.com/cilium/cilium/blob/26dec4c4f4311df2b1a6c909b27ff7fe6e46929f/test/helpers/kubectl.go#L66-L67 (Anchor 'L66-L67' not found)
/src/Documentation/contributing/testing/e2e_legacy.rst:824: WARNING: broken link: https://github.com/cilium/cilium/blob/26dec4c4f4311df2b1a6c909b27ff7fe6e46929f/test/helpers/kubectl.go#L80-L102 (Anchor 'L80-L102' not found)
/src/Documentation/contributing/testing/e2e_legacy.rst:833: WARNING: broken link: https://github.com/cilium/cilium/blob/26dec4c4f4311df2b1a6c909b27ff7fe6e46929f/test/helpers/kubectl.go#L138-L147 (Anchor 'L138-L147' not found)
/src/Documentation/contributing/testing/ci.rst:40: WARNING: broken link: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#organization-member (Anchor 'organization-member' not found)
/src/Documentation/observability/visibility.rst:78: WARNING: broken link: https://github.com/cilium/cilium/tree/master/api/v1/flow#flow-Layer7 (Anchor 'flow-Layer7' not found)
/src/Documentation/contributing/testing/bpf.rst:52: WARNING: broken link: https://github.com/cilium/ebpf/blob/49ebb13083886fc350167f2cde067e094a2b5037/elf_reader.go#L1074 (Anchor 'L1074' not found)
/src/Documentation/contributing/release/backports.rst:314: WARNING: broken link: https://github.com/cilium/release/blob/3c5fc2bdc38f8d290349a612a03cc83655f57a51/pkg/github/labels.go#L26 (Anchor 'L26' not found)
/src/Documentation/security/network/proxy/envoy.rst:17: WARNING: broken link: https://github.com/cilium/proxy#version-compatibility-matrix (Anchor 'version-compatibility-matrix' not found)
/src/Documentation/community/governance/commit_access.rst:210: WARNING: broken link: https://github.com/cncf/foundation/blob/main/README.md#other-content (Anchor 'other-content' not found)

/src/Documentation/network/servicemesh/grpc.rst:61: WARNING: broken link: https://github.com/fullstorydev/grpcurl#binaries (Anchor 'binaries' not found)
/src/Documentation/helm-reference.rst:740: WARNING: broken link: https://github.com/grafana/helm-charts/tree/main/charts/grafana#sidecar-for-dashboards (Anchor 'sidecar-for-dashboards' not found)
/src/Documentation/network/kubernetes/kata.rst:95: WARNING: broken link: https://github.com/kata-containers/packaging/tree/master/kata-deploy#run-a-sample-workload (Anchor 'run-a-sample-workload' not found)
```

Reply:
I don't know :/. I'm travelling today but can take a look into it next week. Please ping me if I forget.

Reply:
OK I had a look, I can reproduce but I don't understand either why these anchors are detected as broken. Maybe the link checker doesn't get exactly what we have in a browser.

So OK let's skip anchors for now, checking the rest of the links would already be a nice improvement.